examples/anime-http/server/asyncapi.yaml

```diff
@@ -0,0 +1,114 @@
+asyncapi: 2.4.0
+info:
+  title: asyncapiImdb server
+  version: 1.0.0
+  description: This app is a dummy server that would stream the trending/upcoming anime.
+servers:
+  trendingAnimeServer:
+    url: http://localhost:8081
+    protocol: http
+channels:
+  trendingAnime:
+    bindings:
+      http:
+        type: request
+        method: POST
+        bindingVersion: 0.1.0
+        query:
+          type: object
+          required:
+            - name
+            - rating
+            - genre
+            - studio
+          properties:
+            name:
+              type: string
+              description: name of the anime.
+            rating:
+              type: string
+              description: rating of the show.
+            genre:
+              type: string
+              description: The genre of anime.
+            studio:
+              type: string
+              description: The studio of anime.
+        body:
+          type: object
+          required:
+            - name
+            - rating
+            - genre
+            - studio
+          properties:
+            name:
+              type: string
+              description: name of the anime.
+            rating:
+              type: number
+              description: rating of the show.
+            genre:
+              type: string
+              description: The genre of anime.
+            studio:
+              type: string
+              description: The studio of anime.
```

Comment:
This is repeated in the client example. Please choose one of them and use `$ref` in the other one.